Packs/SentinelOne/Integrations/SentinelOneV2Mohl/SentinelOneV2Mohl.yml

```diff
@@ -0,0 +1,2277 @@
+category: Endpoint
+commonfields:
+  id: SentinelOne V2 mohl
+  version: -1
+configuration:
```

Comment:
please add the missing `max_fetch` and `first_fetch` configuration parameters (required fetch incidents parameters).